diagnostic.md

```diff
@@ -35,5 +45,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{#each contact.numbers as |number|}}
+    {{my-contact/my-phone number=number}}
+    {{/each}}
```

Comment:
You don't need the `my-contract/`.  
To load the nested component and pass it data, we need to iterate through all of the phone numbers, then we need to access the object property, last we need to bind the sub-component.
```
{{#each contact.phones as |phone|}}
{{my-phone phone=phone}}
{{/each}}
```